src/bbs2gh/Program.cs

```diff
@@ -66,18 +66,18 @@ private static async Task LatestVersionCheck(ServiceProvider sp)
 
             if (await versionChecker.IsLatest())
             {
-                Logger.LogInformation($"You are running the latest version of the ado2gh CLI [v{await versionChecker.GetLatestVersion()}]");
+                Logger.LogInformation($"You are running the latest version of the bbs2gh CLI [v{await versionChecker.GetLatestVersion()}]");
```

Comment:
```suggestion
                Logger.LogInformation($"You are running the latest version of the bbs2gh extension [v{await versionChecker.GetLatestVersion()}]");
```

Reply:
Fixed in b6f8dce93e94a2d605390f0d2975e50b57c52716!  Found another one, too, so I also included it in this commit 👍